starport/pkg/events/events_test.go

```diff
@@ -0,0 +1,191 @@
+package events
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestBus_Send(t *testing.T) {
+	type args struct {
+		e Event
+	}
+	tests := []struct {
+		name string
+		b    Bus
+		args args
```

Comment:
```suggestion
		e Event
```

In `args` type there is only `e`. Since we don't have anything else it can be simplified like this.